### PR TITLE
Incorrect parameter order found by PVS-Studio

### PR DESCRIPTION
**Pull request by DieselMachine84**
I used PVS-Studio static analyzer to check this project. I would like to suggest a variant of the way to fix the error, detected with the help of V3066 diagnostic https://www.viva64.com/en/w/V3066/

### CODE REVIEW

File: src/mscorlib/src/System/Reflection/RuntimeModule.cs [hunk 1]
```diff
@@ -464,7 +464,7 @@ public override Type GetType(String className, bool throwOnError, bool ignoreCas
 
             RuntimeType retType = null;
             Object keepAlive = null;
-            GetType(GetNativeHandle(), className, throwOnError, ignoreCase, JitHelpers.GetObjectHandleOnStack(ref retType), JitHelpers.GetObjectHandleOnStack(ref keepAlive));
+            GetType(GetNativeHandle(), className, ignoreCase, throwOnError, JitHelpers.GetObjectHandleOnStack(ref retType), JitHelpers.GetObjectHandleOnStack(ref keepAlive));
```
Comment (jkotas) [inline src/mscorlib/src/System/Reflection/RuntimeModule.cs]: This is not the right fix. The right fix is to swap the argument names in the GetType declaration at the top of the file. 

There is no actual functionality bug here. It is just a cosmetic issue that the declaration has the argument names swapped.

**Comment by danmosemsft** [comment]
@DieselMachine84 do you have time to fix this so we can merge the pR? thanks for flagging it.

**Comment by DieselMachine84** [comment]
Yes, I'll swap the parameters in declaration

**Comment by jkotas** [comment]
Thanks!

**Pull request merged**